web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -246,16 +417,21 @@ const EncryptionPasswordForm = ({ id, password: passwordProp, onSubmit, onValida
  * @component
  *
  * @param {object} props
- * @param {boolean} props.selected - Whether encryption is selected
- * @param {string} props.password - Password for encryption
- * @param {boolean} props.isLoading - Whether to show the selector as loading
- * @param {onChangeFn} props.onChange - On change callback
+ * @param {string} [props.password=""] - Password for encryption
+ * @param {boolean} [props.isChecked=false] - Whether encryption is selected
+ * @param {boolean} [props.isLoading=false] - Whether to show the selector as loading
+ * @param {onChangeFn} [props.onChange=noop] - On change callback
  *
  * @callback onChangeFn
  * @param {object} settings
  */
-const EncryptionPasswordField = ({ selected: selectedProp, password: passwordProp, isLoading, onChange }) => {
-  const [selected, setSelected] = useState(selectedProp);
+const EncryptionPasswordField = ({
+  password: passwordProp = "",
```

Comment:
NP: You could call it `value`, by mimicking the prop of other TextInputs. Of course, it is `password` internally.